common/src/main/java/org/dash/wallet/common/data/CurrencyInfo.java

```diff
@@ -60,15 +60,20 @@
 
     static {
         obsoleteCurrencyMap = new HashMap<>();
-        obsoleteCurrencyMap.put("BYR", "BYN");
+        obsoleteCurrencyMap.put("BYR", "BYN"); // Belarus Ruble, changed in 2016
+        obsoleteCurrencyMap.put("MRO", "MRU"); // Mauritania Ouguiya, changed in 2018
 
         otherCurrencyMap = new HashMap<>();
         otherCurrencyMap.put("VES", "Venezuelan Bolívar");
         otherCurrencyMap.put("GGP", "Guernsey Pound");
         otherCurrencyMap.put("JEP", "Jersey Pound");
         otherCurrencyMap.put("IMP", "Isle of Man Pound");
         otherCurrencyMap.put("USDC", "USD Coin");
-
+        otherCurrencyMap.put("LTC", "Litecoin");
+        otherCurrencyMap.put("ETH", "Etherium");
+        otherCurrencyMap.put("BTC", "Bitcoin");
+        otherCurrencyMap.put("PAX", "Paxos Standard");
+        otherCurrencyMap.put("GUSD", "Gemini Dollar");
```

Comment:
Will not be used I suppose? https://github.com/dashevo/dash-wallet/pull/298/commits/afc6bd3719960f8810489950a47e0c3b0026e5ec#diff-d59db2bdbe7841e4cb222c288bc4e156R76

Reply:
There is one currency, USDT that will be 4 letters long.  This check will prevent the exception that occurs for currency codes that are not 3 letters.